scripts/fledge

```diff
@@ -22,7 +22,7 @@ set -e
 #
 # This is the startup script for fledge
 #
-USAGE="Usage: `basename ${0}` {start|start --safe-mode|stop|status|reset|purge|kill|healthcheck|help|version}"
+USAGE="Usage: `basename ${0}` <-h> <-u username> <-p password> <-c certificate> {start|start --safe-mode|stop|status|reset|purge|kill|healthcheck|help|version}"
```

Comment:
These flags are primarily used for cases like stop and health checks, and they might cause confusion in other scenarios.